README.md

```diff
@@ -9,21 +9,27 @@ https://github.com/mongodb/docs-java
 
 ## Build Instructions
 
+:warning: **When building docs for a new version, make sure to update the following:**
+- `<this repo>/reference/config.toml` to point to the new base version
+- `<this repo>/landing/data/releases.toml` to link to the docs for the new version
+- any redirects such as the [Java sync docs redirect](https://github.com/mongodb/docs-java-other/pull/3/files#diff-0f1a8692163867f83ff7451f3018bae71d3d16dbee396abf03263784e5dda940)
```

Comment:
Issue: I think the first element of the list item should be [capitalized](https://docs.mongodb.com/meta/style-guide/style/lists/#list-items)

```suggestion
- Any redirects such as the [Java sync docs redirect](https://github.com/mongodb/docs-java-other/pull/3/files#diff-0f1a8692163867f83ff7451f3018bae71d3d16dbee396abf03263784e5dda940)
```